paging/paging-testing/build.gradle

```diff
@@ -15,24 +15,39 @@
  */
 
 import androidx.build.LibraryType
+import androidx.build.PlatformIdentifier
 import androidx.build.Publish
 
 plugins {
     id("AndroidXPlugin")
-    id("com.android.library")
-    id("org.jetbrains.kotlin.android")
```

Comment:
I think we need to continue targeting android because the first stable version is to be released with android support. @ianhanniballake What are your thoughts on this?

Reply:
Given that these are testing paging-common APIs, I think it makes to match the same set of targets.

Reply:
@veyndan We are in the process of refactoring Logger in paging-common with added android target so paging-common will become an android library. Working on the CL as we speak, should be up today. I'll cc you on it.

Reply:
I've [updated paging-common](https://android-review.googlesource.com/c/platform/frameworks/support/+/2651248/4) into an android library now.

Reply:
Just updated the PR to add back the android target!